common/src/main/java/com/wynntils/core/features/overlays/TextOverlay.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.overlays;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.features.overlays.sizes.OverlaySize;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.buffered.BufferedFontRenderer;
+import com.wynntils.utils.render.type.HorizontalAlignment;
+import com.wynntils.utils.render.type.TextShadow;
+import com.wynntils.utils.render.type.VerticalAlignment;
+import net.minecraft.client.renderer.MultiBufferSource;
+
+/**
+ * An overlay, which main purpose is to display function templates.
+ */
+public abstract class TextOverlay extends Overlay {
+    @Config
+    public TextShadow textShadow = TextShadow.OUTLINE;
```

Comment:
I forgot how inherited configs work. Do you need to update i18n for these? How does it work when we have multiple overlays inheriting this?

But I'm guessing at least some i18n changes is needed for this patch to work..?

Reply:
Fixed it up, but it is only needed for the future.